dnf-docker-test/features/provides-1.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Test for dnf provides
+
+  @setup
+  Scenario: Setup (create test repo)
+      Given repository "base" with packages
+         | Package      | Tag       | Value         |
+         | TestA        | Version   | 1             |
+         |              | Requires  | TestB         |
+         | TestC        | Version   | 1             |
+         | TestD        | Version   | 1             |
+         |              | Provides  | TestB         |
+         | TestE        | Version   | 1             |
+         |              | Provides  | TestEVal      |
+        And repository "update" with packages
+         | Package      | Tag       | Value         |
+         | TestA        | Version   | 2             |
+         |              | Requires  | TestB         |
+         | TestC        | Version   | 2             |
+         |              | Provides  | TestB         |
+         |              | Conflicts | TestD         |
+         | TestD        | Version   | 2             |
+         |              | Provides  | TestB         |
+         |              | Conflicts | TestC         |
+         | TestE        | Version   | 2             |
+         |              | Provides  | TestEVal      |
+       When I enable repository "base"
+        And I successfully run "dnf -y install TestA TestC TestD"
+        And I disable repository "base"
+
+  Scenario: dnf provides TestB - installed package TestD-1 provides TestB
+       When I successfully run "dnf provides TestB"
+       Then the command stdout should match regexp "TestD-1-[^R]+Repo[ \t]+: @System"
```

Comment:
1. The order of printing of packages is not defined. When I run tests more times the order differs.
2. Want we strictly test number of spaces (etc.)?